google/cloud/spanner_v1/snapshot.py

```diff
@@ -55,6 +60,17 @@ def _restart_on_unavailable(restart, trace_name=None, session=None, attributes=N
             with trace_call(trace_name, session, attributes):
                 iterator = restart(resume_token=resume_token)
             continue
+        except InternalServerError as exc:
+            resumable_error = any(
+                resumable_message in exc.message
+                for resumable_message in _STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES
```

Comment:
`resumable_error = exc.message in _STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES`? 

Even better: 

```python
if exc.message not in _STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES:
    raise
...
```

Reply:
I believe the intention here is that `_STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES` contains error substrings. This mean we don't need to include multiple entries for `RST_STREAM` errors. I know we haven't encountered those errors in Spanner yet but I think it's worth including since the rationale appears to be the same.

Reply:
Okay. I see your point.